views/timeline/index.hbs

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link title="timeline-styles" rel="stylesheet" href="https://cdn.knightlab.com/libs/timeline3/latest/css/timeline.css">
+    <script src="https://cdn.knightlab.com/libs/timeline3/latest/js/timeline.js"></script>
+</head>
+
+<body>
+    <div id='timeline-embed' style="width: 100%; height: 600px"></div>
+    <script type="text/javascript">
+        fetch('/timeline/latestData')
+            .then(function (response) {
+                return response.json();
+            }).then(function (myJson) {
```

Comment:
can we avoid var names like `myJson`?, `data` or `JSONdata` might be more appropriate.